pylops/signalprocessing/_fourierradon2d_numba.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+import numpy as np
+from numba import jit, prange
+
+# detect whether to use parallel or not
+numba_threads = int(os.getenv("NUMBA_NUM_THREADS", "1"))
+parallel = True if numba_threads != 1 else False
+
+
+@jit(nopython=True, parallel=parallel, nogil=True, cache=True, fastmath=True)
+def _radon_inner_2d(X, Y, f, px, h, flim0, flim1, npx, nh):
+    for ih in prange(nh):
+        for ifr in range(flim0, flim1):
+            for ipx in range(npx):
+                Y[ih, ifr] += X[ipx, ifr] * np.exp(
+                    -1j * 2 * np.pi * f[ifr] * px[ipx] * h[ih]
+                )
```

Comment:
why not use cmath.exp and math.pi like in the cuda version? same for the 3d implementation

Reply:
Ops, that's just a 'mistake'... I changed it now :)